MusicApp.PlaylistService.Web/MessageBroker/SongConsumer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MassTransit;
+using MusicApp.PlaylistService.Application.Services.Interfaces;
+using MusicApp.Shared;
+
+namespace MusicApp.PlaylistService.Web.MessageBroker;
+
+public class SongConsumer : IConsumer<SongMessage>
+{
+    private readonly ISongService _service;
+
+    public SongConsumer(ISongService service)
+    {
+        _service = service;
+    }
+
+    public async Task Consume(ConsumeContext<SongMessage> context)
+    {
+        switch(context.Message.Operation)
+        {
+            case Operation.Created:
+                await _service.AddSongAsync(context.Message.Song, new CancellationToken());
+                break;
+
+            case Operation.Updated:
+                await _service.UpdateSongAsync(context.Message.Song, new CancellationToken());
+                break;
+
+            case Operation.Deleted:
+                await _service.RemoveSongAsync(context.Message.Song, new CancellationToken());
+                break;
+        }
+
```

Comment:
extra line

Reply:
I removed everything related to song messages. And now playlist service can get songs via grpc.